docs/docs/howto/vault.md

```diff
@@ -1,224 +1,161 @@
+spec:
+  template:
+    metadata:
+      annotations:
+        contrast.edgeless.systems/workload-secret-id: vault_unsealing
 ```
 
-</TabItem>
-</Tabs>
+Vault's TCP listener is configured to use the certificate provisioned by the initializer.
+Below is the relevant excerpt from the configuration file.
 
-### Download the Contrast Coordinator resource
-
-Download the Kubernetes resource of the Contrast Coordinator, comprising a single replica deployment and a LoadBalancer service.
-Put it next to your resources:
-
-```sh
-curl -fLO https://github.com/edgelesssys/contrast/releases/latest/download/coordinator.yml --output-dir deployment
+```hcl
+listener "tcp" {
+  address            = "0.0.0.0:8200"
+  tls_cert_file      = "/contrast/tls-config/certChain.pem"
+  tls_key_file       = "/contrast/tls-config/key.pem"
+}
 ```
 
-### Generate policy annotations and manifest
-
-Run the `generate` command to generate the execution policies and add them as annotations to your deployment files.
-A `manifest.json` file with the reference values of your deployment will be created:
+The Vault configuration file would ideally just be mounted from the `ConfigMap` into the server pod.
+However, this would be insecure, as the Vault configuration is sensitive but mounts aren't integrity-checked (see [Hardening](hardening.md#limitations-inherent-to-policy-checking)).
+Therefore, the demo pod adds the configuration as an environment variable and extracts it to a memory-backed file before starting the server.
 
-<Tabs queryString="platform">
-<TabItem value="aks-clh-snp" label="AKS" default>
-
-```sh
-contrast generate --reference-values aks-clh-snp deployment/
-```
-
-</TabItem>
-<TabItem value="k3s-qemu-snp" label="Bare metal (SEV-SNP)">
-
-```sh
-contrast generate --reference-values k3s-qemu-snp deployment/
-```
+### Deploy Vault
 
-:::note[Missing TCB values]
+Follow the steps of the generic workload deployment instructions:
 
-On bare-metal SEV-SNP, `contrast generate` is unable to fill in the `MinimumTCB` values as they can vary between platforms.
-They will have to be filled in manually.
-If you don't know the correct values use `{"BootloaderVersion":255,"TEEVersion":255,"SNPVersion":255,"MicrocodeVersion":255}` and observe the real values in the error messages in the following steps.
-This should only be done in a secure environment.
-Note that the values will differ between CPU models.
+- [Add the Coordinator.](workload-deployment/add-coordinator.md)
+- [Generate the policies.](workload-deployment/generate-annotations.md)
+  - After running `contrast generate`, add the desired Vault domain name to the `SANs` array in `manifest.json`.
+- [Apply the resources.](workload-deployment/deploy-application.md)
+- Configure your Vault domain name to resolve to the load balancer IP of the Vault service.
+  Alternatively, configure the system from which you access Vault to resolve `vault` to the load balancer IP.
+  On Linux, you can modify `/etc/hosts` for this.
```

Comment:
Much clearer IMO, but still: 
- this is the only place in the doc page where `load balancer` appears, and searching the entire docs isn't helpful, either
- similar to the next linked step, it could be helpful to provide
  ```
  kubectl get svc vault -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
  ```

Reply:
Moved this into a section of its own, because it got a bit large for a bullet point.